src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -24,6 +24,22 @@
 }
 
 .osdQueryEditor__dataSourceWrapper {
+  .dataSourceSelect {
+    border-bottom: $euiBorderThin !important;
+
+    :first-child {
+      box-shadow: none !important;
+      height: 100%;
+      border-radius: 0;
+    }
+
+    div:is([class$="--group"]) {
+      padding: 0 !important;
```

Comment:
i see `!important` is used in many places already, is it avoidable?

Reply:
it's actually the OUI combo box. I would like to avoid it but some of the classes everywhere have like added padded so this is to avoid the height of the flex box getting pushed around